tests/test_cache_utils.py

```diff
@@ -293,7 +293,7 @@ def test_sink_cache_iterative_prompts(self):
     @parameterized.expand(["eager", "sdpa", "flash_attention_2"])
     def test_static_cache_greedy_sampling_pad_left(self, attn_implementation):
         EXPECTED_GENERATION = [
-            "The best color is the one that complements the subject you are photograph",
+            "The best color is the one that complements the skin tone of the",
```

Comment:
These changed test results were checked against `4b236aed7618d90546cd2e8797dab5b4a24c5dce` (the commit before the static caches were introduced).

These tests do batched generation, hence the need to change.